src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
I'd prefer `generateAddress` over `generateRippleAddress`. I don't think there is any ambiguity because there is only one type of address in the API.